types/glob-parent/glob-parent-tests.ts

```diff
@@ -0,0 +1,4 @@
+import * as globParent from 'glob-parent';
```

Comment:
Since `globParent` is a function, and an ES6 module object cannot be invoked as a function or with new(https://stackoverflow.com/questions/39415661/what-does-resolves-to-a-non-module-entity-and-cannot-be-imported-using-this), so it is preferred to use `import globParent = require('glob-parent');` and then the namespace can be removed.